core/mondoo-kubernetes-best-practices.mql.yaml

```diff
@@ -1999,10 +1999,15 @@ queries:
     remediation: |
       For all Secrets with expired or soon-to-expire certificates, update the Secret data with refreshed certificates.
   query: |
-    k8s.ingress.tls{
-      certificates {
-        expiresIn.days > 15
-      }
+    len = k8s.ingress.tls.length == 0
+    if( len == 0 ){
```

Comment:
do we need to check for len > 0?
is all of an empty list not true?

Reply:
it absolutely is. i'll update it to clean up the query